packages/http-server-fastify/src/FastifyHttpServer.ts

```diff
@@ -27,10 +27,8 @@ import qs from 'qs';
 type Server = HttpServer | HttpsServer;
 
 export interface FastifyHttpServerModuleOptions extends HttpServerModuleOptions {
-	instance?: FastifyInstance;
-	middlewares?: {
```

Comment:
keeping this nesting could be beneficial to maintain a similar structure as the [ExpressHttpServer module](https://github.com/HPInc/davinci/blob/master/packages/http-server-express/src/ExpressHttpServer.ts#L24).

Also, we should be careful not to introduce breaking changes too often

Reply:
My idea was to remove the middleware concept from fastify